eva-cli/src/main.rs

```diff
@@ -125,18 +125,28 @@ fn dispatch(inputs: &ArgMatches, configuration: &Configuration) -> Result<()> {
             Ok(set_field(configuration, field, id, value)?)
         }
         ("tasks", Some(_submatches)) => {
-            let tasks = block_on(eva::tasks(configuration))?;
-            println!("Tasks:");
```

Comment:
Didn't you like the "Tasks:" header? Or did you remove it by accident?

Reply:
By accident :P